plutus-tx-plugin/src/PlutusTx/Compiler/Type.hs

```diff
@@ -236,10 +236,23 @@ getDataCons tc' = sortConstructors tc' <$> extractDcs tc'
           | GHC.isFamilyTyCon tc = throwSd UnsupportedError $ "Irreducible type family application:" GHC.<+> GHC.ppr tc
           | otherwise = throwSd UnsupportedError $ "Type constructor:" GHC.<+> GHC.ppr tc
 
+{- Note [On workers and wrappers]
```

Comment:
Maybe "On data constructor workers and wrappers", since "worker" and "wrapper" come up in lots of other contexts too.